packages/contracts/contracts/libraries/ProjectHelpers.sol

```diff
@@ -1,22 +1,26 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.0 <0.9.0;
 
-import {IController} from "../discovery/IController.sol";
+import {Project} from "../discovery/Project.sol";
```

Comment:
this is no longer used, is it?

Reply:
we talked about making this more generic and with more descriptive names

I would leave it for now to see if the use case appears. If it doesn't, we can then remove it

Reply:
what's the point of that though?
I would argue that if a function depends on the project contract, it would probably just be a contract function, not a library one

Reply:
sorry, I thought you were talking about the entire library

you are absolutely right, this import is no longer needed